File: appJava/src/main/java/game/Message.java
package game;

import org.json.simple.JSONObject;

public class Message {
    int idMatch;
    int turn;
    ePiece adversaryPlay;

    public Message(int _idMatch, int _turn, ePiece _adversaryPlay)
    {
        idMatch = _idMatch;
        turn = _turn;
        adversaryPlay = _adversaryPlay;
    }
    
    public JSONObject createJsonMessage()
    {
        JSONObject json = new JSONObject();
        json.put("idMatch", idMatch);
        json.put("turn", turn);
        json.put("adversaryPlay", adversaryPlay.toString());
        return json;
    }

    

    
}

File: appPython/Message.py
from ePiece import ePiece
import json

class Message():

    def __init__(self,idMatch: int, turn: int,adversaryPlay: ePiece):
        self.idMatch = idMatch
        self.turn = turn
        self.adversaryPlay = adversaryPlay
    
    def createJsonMessage(self):
        dict = {"idMatch": self.idMatch,
                "turn": self.turn,
                "adversaryPlay": self.adversaryPlay.name}
        return json.dumps(dict)